### ISSUE TITLE
Missing CriticalHandle*

### ISSUE THREAD

**Issue by danmosemsft** [dotnet/corefx#11729]
These are in NS 2.0 and are missing from Core:

```
T:Microsoft.Win32.SafeHandles.CriticalHandleMinusOneIsInvalid
M:Microsoft.Win32.SafeHandles.CriticalHandleMinusOneIsInvalid.#ctor
P:Microsoft.Win32.SafeHandles.CriticalHandleMinusOneIsInvalid.IsInvalid
M:Microsoft.Win32.SafeHandles.CriticalHandleMinusOneIsInvalid.get_IsInvalid
T:Microsoft.Win32.SafeHandles.CriticalHandleZeroOrMinusOneIsInvalid
M:Microsoft.Win32.SafeHandles.CriticalHandleZeroOrMinusOneIsInvalid.#ctor
P:Microsoft.Win32.SafeHandles.CriticalHandleZeroOrMinusOneIsInvalid.IsInvalid
M:Microsoft.Win32.SafeHandles.CriticalHandleZeroOrMinusOneIsInvalid.get_IsInvalid
```

**Comment by davidsh** [comment]
>M:Microsoft.Win32.SafeHandles.CriticalHandleZeroOrMinusOneIsInvalid.#ctor

System.Net currently created an internal version of this which will need to be removed when we do this work.

**Comment by kouvel** [comment]
Updated inline adding a few more related ones to the list. Issue #11299 tracks removal of the internal safe handle types, I'll add these critical handle types to that as well.

**Comment by kouvel** [comment]
Fixed by #12392
